doc/openms/_static/css/custom.css

```
@@ -0,0 +1,152 @@
+    align-items: center;
+}
+
+img.halfwidth {
+    width: 50%;
+    margin-top: 10px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+
+/*add numbers to headings*/
```

Stylelint (17.6.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)